annotation/tests/test_tasks_services.py

```diff
@@ -0,0 +1,158 @@
+from unittest.mock import Mock, call, patch
+
+from annotation.models import File
+import pytest
+from sqlalchemy.orm import Session
+
+from annotation.errors import FieldConstraintError
+from annotation.jobs.services import ValidationSchema
+from annotation.tasks.services import validate_files_info, validate_task_info, validate_users_info
+
+
+TEST_PARAMS_INVALID_FILES = [
+    (
+        {
+            "file_id": 1,
+            "job_id": 2,
+            "pages": [1, 2, 4],
+        },
+        Mock(spec=File, pages_number=3),
+        "pages ({4}) do not belong to file"
+    ),
+    (
+        {
+            "file_id": 1,
+            "job_id": 2,
+            "pages": [1, 2, 3],
+        },
+        None,
+        "file with id 1 is not assigned for job 2"
+    ),
+]
+
+
+@pytest.fixture(scope="function")
```

Comment:
Default scope for fixture is function, no need to write it here
```suggestion
@pytest.fixture
```

Reply:
Fixed :)